great_expectations/experimental/datasources/spark_datasource.py

```diff
@@ -49,7 +48,6 @@ class _SparkDatasource(Datasource):
     asset_types: ClassVar[List[Type[DataAsset]]] = [CSVSparkAsset]
 
     # instance attributes
-    type: str = pydantic.Field("_spark")
```

Comment:
@Kilo59 I may need some help next week figuring out the right way to create `SparkFilesystemDatasource`, `SparkS3Datasource`, and so on in terms of the `type` field as well as the `Schema` file naming and generation.  Thank you.

Reply:
I would think this should just be `'spark_file'` or `'spark_filesystem'` and `'spark_s3'`.